backend/api/views/simplePostView.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework import status
+from django.core import serializers
+
+from ..models import post
+from ..serializers import PostSerializer
+
+import json, sys
+#########################################
+# post request, asking for 
+#    title, str
+#    author_id, uuid
+#    description, str
+#    content, str
+#    visibility, str
+#
+#
+##########################################
+@api_view(['POST'])
+def postAuthorPost(request, author_id):
```

Comment:
I feel like contentType should also be sent in the POST, because that way we don't have to figure it out on the backend, what do you think?

Reply:
Added into the serializer, so if user gave `contentType` the field will update

Reply:
One last question abot the naming of the views. I am not sure if it is a big deal but since this is a view pertaining only to `Posts` could calling the method `postAuthorPost` be confusing? If you think its better this way im fine with it, I just want a convention to go with

Reply:
`postSubmittedPost`? IDK, any idea with naming? It is not a skill I am very confident with